ansible/roles/stack-sunbird/templates/search-service_application.conf

```diff
@@ -346,4 +346,4 @@ compositesearch.index.name="{{ compositesearch_index_name }}"
 
 content.tagging.backward_enable=true
 content.tagging.property=["subject","medium"]
-search.payload.log_enable=true
+search.payload.log_enable=false
```

Comment:
I think this should be variable. 
```
 search.payload.log_enable={{search_payload_log_enabled|d(false)}}
```

Reply:
Hi Rajesh, it is wrong property. THis property was not expected to change. It was just confusion.

Reply:
It was wrong property, sorry.